src/renderer/components/RetypePasswordForm.tsx

```diff
@@ -162,29 +220,31 @@ const RetypePasswordForm = ({
           {passwordConfirm.length > 0 && strengthHint(passwordConfirm)}
         </FormHelperText>
       </FormControl>
-      {useActivationKey
-        ? (
-          <FormControl
-            fullWidth
-            error={isActivationKeyError}
-            className={classes.formControl}
-          >
-            <InputLabel className={classes.label}>{locale("초대 코드")}</InputLabel>
-            <OutlinedInput type="text" onChange={handleActivationKeyChange} />
-            <FormHelperText className={classes.helperText}>
-              form helper text
-            </FormHelperText>
-          </FormControl>
-        )
-        : (<></>)}
+      {useActivationKey ? (
+        <FormControl
+          fullWidth
+          error={isActivationKeyError}
+          className={classes.formControl}
+        >
+          <InputLabel className={classes.label}>
+            <T _str="Invitation Code" _tags="retypePassword" />
+          </InputLabel>
+          <OutlinedInput type="text" onChange={handleActivationKeyChange} />
+          <FormHelperText className={classes.helperText}>
+            form helper text
+          </FormHelperText>
+        </FormControl>
+      ) : (
+        <></>
+      )}
```

Comment:
Maybe we can use `&&` condition render instead `?`

```suggestion
      {useActivationKey && (
        <FormControl
          fullWidth
          error={isActivationKeyError}
          className={classes.formControl}
        >
          <InputLabel className={classes.label}>
            <T _str="Invitation Code" _tags="retypePassword" />
          </InputLabel>
          <OutlinedInput type="text" onChange={handleActivationKeyChange} />
          <FormHelperText className={classes.helperText}>
            form helper text
          </FormHelperText>
        </FormControl>
      )}
```

Reply:
Uh, actually I think we gotta avoid doing that. See: https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx

Reply:
How about use `!!useActivationKey &&` ?